### PR TITLE
Add yaml metadata for metrics.csv files

### PR DESCRIPTION
**Pull request by Ancairon**
##### Summary
This PR adds the info that the CSV files contain in yaml format, additional user action is needed to fill in the rest of the yaml template, that will be discussed internally. This is the auto-generated part of the yaml that we need.

##### Additional Information

<details> <summary>For users: How does this change affect me?</summary>
  
</details>

### CODE REVIEW

**Comment by Ancairon** [comment]
Yamls should have a level 2 indentation as more info will follow on the top of each one, so the yaml lint check is expected to fail

File: collectors/charts.d.plugin/apcupsd/metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
+  - metrics:
+      module:
+        name: apcupsd
```
Comment (ilyam8) [inline collectors/charts.d.plugin/apcupsd/metadata.yaml:3]: Why this structure? It doesn't reflect the one we have in netdata/go.d/plugin/modules/*.

File: collectors/proc.plugin/metadata.yaml [hunk 2]
```diff
@@ -0,0 +1,2298 @@
+  - metrics:
+      module:
+        name: /proc/stat
```
Comment (ilyam8) [inline collectors/proc.plugin/metadata.yaml:3]: Why module.name is inside metrics?

**Comment by andrewm4894** [comment]
@Ancairon is it possible to just add empty sections for the parts to be manually updated later? 

what would be an example?

**Comment by Ancairon** [comment]
@andrewm4894 I thing it would make generation more complex + some stuff would be as you say, empty, you would still need to look how we do it

example: https://github.com/netdata/go.d.plugin/blob/master/modules/apache/metadata.yaml

**Comment by andrewm4894** [comment]
@Ancairon but why would `name` and `title` not be in the files in this PR?

Would it not be better to spec it all out and even put dummy strings in the PR for now of stuff we need to clean up later so we can force ourselves to do it and make it easier to get a bigger picture on what the end state is here?

Either we know enough at this stage (or as part of discussing this PR) to at least add a minimal full examples for each - even if has some dummy filler text in places - at least then we are a bit clearer on what finished or bigger picture is here. 

> ...as more info will follow on the top of each one...

^ its this i think i would like to fully understand as part of reviewing this PR as i don't think i can assume its independent. e.g. for sure if we wanted to add new sections later in follow on PR's for some new stuff we need then that makes sense. But i think would be good to try have this PR be a full minimal one that we (as best we know right now) think we need.

**Comment by Ancairon** [comment]
I spotted some duplicates on some of the generated alerts, will do some debugging to try and fix this. (for example python.d.plugin/beanstalk)

**Review by andrewm4894 — APPROVED** [review]

**Pull request merged**